get-started/installation.mdx

```diff
@@ -35,6 +35,42 @@ const literalClient = new LiteralClient(process.env['LITERAL_API_KEY']); // This
 
 To get your API key, go to your [project page](https://cloud.getliteral.ai) and click on the **Settings** tab. You will find your API key in the **API Key** section.
 
+## Log your first thread
+
+<CodeGroup>
+```python Python
+import os
+from literalai import LiteralClient
+
+# Not compatible with gunicorn's --preload flag
+literal_client = LiteralClient(api_key=os.getenv("LITERAL_API_KEY")) # This is the default and can be omitted
```

Comment:
Let's avoid those comments, it looks confusing to me

Reply:
you're right they were pasted over from the previous block but are not necessary here